doc/nrf/ug_thread_configuring.rst

```diff
@@ -232,6 +232,21 @@ Minimal Thread Device (MTD)
   By default, the MTD operates as Minimal End Device (MED).
   To make it operate as Sleepy End Device (SED), set :kconfig:`CONFIG_OPENTHREAD_MTD_SED`.
 
+.. _thread_ug_tfm_support:
+
+Trusted Firmware-M support options
+==================================
+
+To configure your Thread application on the nRF5340 DK to run with Trusted Firmware-M, enable the following Kconfig options:
+
+* :kconfig:`CONFIG_BUILD_WITH_TFM`
+* :kconfig:`CONFIG_OPENTHREAD_CRYPTO_PSA`
+
+These options are enabled by default for the :ref:`application core <ug_nrf5340_intro_app_core>` of the ``nrf5340dk_nrf5340_cpuapp_ns`` build target.
```

Comment:
They are enabled by default for this build target, but only in the case of OT samples included in NCS. Not sure if that's what you meant.

Reply:
Rephrased slightly.